setup.py

```diff
@@ -27,7 +27,6 @@
     classifiers=[
         'Environment :: Plugins',
         'Framework :: Django',
-        "Framework :: Django :: 3.2",
```

Comment:
Shouldn't you drop the 4.0 and 4.1 classifiers too?

Reply:
Good catch! Yes, removed them.